server/sql_app/models.py

```diff
@@ -98,30 +98,27 @@ class Pattern(Base):
     owner = relationship("Session", back_populates="patterns")
 
 
+# TODO: It may make sense for this table to be global,
+# rather than in context of a Session/User.
 class ParcelStats(Base):
     """SQLAlchemy model for storing lulc stats under parcels."""
     __tablename__ = "parcel_stats"
 
-    stats_id = Column(Integer, primary_key=True, index=True)
+    stats_id = Column(Integer, index=True, primary_key=True)
+    parcel_id = Column(Integer)
     target_parcel_wkt = Column(String)
     lulc_stats = Column(String)
-    #TODO: I'm not sure if parcel stats not associated with a scenario
-    # should be related to another table...
     job_id = Column(Integer, ForeignKey("jobs.job_id"))
 
-    #owner = relationship("Job", back_populates="parcel_stats")
-
 
 class Parcel(Base):
     """SQLAlchemy model for parcels."""
     __tablename__ = "parcel"
 
-    id = Column(Integer, primary_key=True, index=True)
+    study_area_id = Column(String, ForeignKey("study_area.id"), primary_key=True)
+    parcel_id = Column(Integer, ForeignKey("parcel_stats.parcel_id"), primary_key=True)
     wkt = Column(String)
-    lulc_stats = Column(String)
     address = Column(String)
 
-    # each scenario has an associated study area owner
-    study_area_id = Column(String, ForeignKey("study_area.id"))
-
     study_area = relationship("StudyArea", back_populates="parcels")
+    parcel_stats = relationship("ParcelStats", uselist=False)
```

Comment:
We were still doing this relationship wrong. In this case, we don't want `back_populates because we do not want the `parcel_stats` table to update itself after changes (such as row deletion) from the `parcel` table. `parcel_stats` should always hold the LULC stats for a parcel, even if it is has been removed from all study areas. Also, I believe we were defining the ForeignKey backwards, though I'm not actually certain of that.